### PR TITLE
Ho 16 create a listing component

### PR DESCRIPTION
**Pull request by tlchambers**

### CODE REVIEW

File: frontend/src/components/newRequest/NewRequest.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
+import React, { useState } from "react";
+
+const NewRequestForm = ({ navigate }) => {
+  const [organisationName, setorganisationName] = useState("");
```
Comment (Sabirah42) [inline frontend/src/components/newRequest/NewRequest.js:4]: Naming after 'set' shoulud be standardised- some are camel case and others are a mix

File: frontend/src/components/newRequest/NewRequest.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  const [city, setCity] = useState("");
+  const [postcode, setPostcode] = useState("");
+  const [neededByDate, setneededByDate] = useState("");
+  const [error, setError] = useState(null);
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    // '/users' to be amended when we have clarity of route path for backend
+    const response = await fetch("/users", {
```
Comment (Sabirah42) [inline frontend/src/components/newRequest/NewRequest.js:18]: route should be '/api/orgUsers'

File: frontend/src/components/newRequest/NewRequest.js [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+            <label id="form_label" htmlFor="first-line">
+              First Line of Address
+            </label>
+            <input
+              className="form_field"
+              id="first-line"
+              type="text"
+              value={firstLine}
+              onChange={handleFirstLineChange}
+              placeholder="2 Redman Place"
```
Comment (Sabirah42) [inline frontend/src/components/newRequest/NewRequest.js:155]: Should put 'e.g.' in front of the placeholders

File: frontend/src/components/orgSignUp/OrgSignUpForm.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react';
-import './OrgsSignUpForm.css';
+// import './OrgsSignUpForm.css';
```
Comment (Sabirah42) [inline frontend/src/components/orgSignUp/OrgSignUpForm.js:2]: Should this be singular? Has been changed to singular in your first file (App.js).
We've used singular for our naming as well

File: frontend/src/components/newRequest/NewRequest.js (hunk 1 above)
Comment (tlchambers) [reply frontend/src/components/newRequest/NewRequest.js]: Thanks this has now been amended

File: frontend/src/components/orgSignUp/OrgSignUpForm.js (hunk 4 above)
Comment (tlchambers) [reply frontend/src/components/orgSignUp/OrgSignUpForm.js]: Yes we missed a few when we were amending the names to match backend, this has now been actioned.

**Review by mantaslv — CHANGES_REQUESTED** [review]
small change from my side for the route of the fetch, app should be all connected after this!! :)

File: frontend/src/components/newRequest/NewRequest.js [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+  const [city, setCity] = useState("");
+  const [postcode, setPostcode] = useState("");
+  const [neededByDate, setNeededByDate] = useState("");
+  const [error, setError] = useState(null);
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    // '/users' to be amended when we have clarity of route path for backend
+    const response = await fetch("/users", {
```
[the remark this answers is not in the document]
Comment (tlchambers) [reply frontend/src/components/newRequest/NewRequest.js]: I have now actioned this. I have updated route path for signup; login and listing 👍

File: frontend/src/components/newRequest/NewRequest.js (hunk 3 above)
Comment (tlchambers) [reply frontend/src/components/newRequest/NewRequest.js]: I put in placeholders for the time being but that is something we can discuss later on when we do styling as all the placeholders isn't appealing to the :face_exhaling:

**Review by mantaslv — APPROVED** [review]

**Pull request merged**